src/Tools/MicroBuild/Build.proj

```diff
@@ -25,6 +25,8 @@
 
     <MSBuild Projects="$(ProjectDir)Roslyn.sln" BuildInParallel="true" Properties="DeployExtension=false" />
 
+    <Exec Command="$(NuGetPackageRoot)\GitLink\$(GitLinkVersion)\lib\net45\GitLink.exe $(ProjectDir) -u https://github.com/dotnet/roslyn -f Roslyn.sln -c $(Configuration)" />
```

Comment:
Are we still tracking the "make this run for PRs once we fix the perf issues"? I wholeheartedly approve this and don't want to block, but still want to make sure we're tracking the improvement _somewhere_.